inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/iceberg/IcebergSinkDTO.java

```diff
@@ -63,15 +60,6 @@ public class IcebergSinkDTO {
     @ApiModelProperty("File format, support: Parquet, Orc, Avro")
     private String fileFormat;
```

Comment:
suggest use enum `org.apache.inlong.manager.common.enums.FileFormat`

Reply:
good idea, will consider in another pr.